.github/workflows/continuous-deployment.yml

```diff
@@ -259,7 +258,8 @@ jobs:
           npm run build
 
       - name: Execute Tests
-        run: npm run test
+        run: |
+          export APPINSIGHTS_CONNECTION_STRING="TESTSTRING" && npm run test
```

Comment:
This change my puzzle some people. For running the tests on the service we do not need a valid connection string in the environment variables for application insights, but it does need a non null value in order for them to pass. In order to prevent any collection from the App Insights resources, we pass in an invalid string for the test components to run when the functions are started.

Reply:
@jamesobrooks  Yes it would. That is a good pickup.